data_steward/tools/retract_data_gcs.py

```diff
@@ -159,32 +171,55 @@ def get_integer(num_str):
         return None
 
 
+def extract_pids_from_table(project_id, sandbox_dataset_id, pid_table_id):
+    """
+    Extracts person_ids from table in BQ in the form of a set of integers
+
+    :param project_id: project containing the sandbox dataset with pid table
+    :param sandbox_dataset_id: dataset containing the pid table
+    :param pid_table_id: identifies the table containing the person_ids to retract
+    :return: set of integer pids
+    """
+    q = EXTRACT_PIDS_QUERY.format(project_id=project_id,
+                                  sandbox_dataset_id=sandbox_dataset_id,
+                                  pid_table_id=pid_table_id)
+    r = bq_utils.query(q)
+    rows = bq_utils.response2rows(r)
+    pids = set()
+    for row in rows:
+        pids.add(get_integer(row['person_id']))
+    return pids
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter)
```

Comment:
Please say what the command does in `ArgumentParser.description`.

Reply:
Done